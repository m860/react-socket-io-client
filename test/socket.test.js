/**
 * @overview 文件描述
 * @author jean.h.ma
 */