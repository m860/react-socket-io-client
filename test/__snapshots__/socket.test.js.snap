// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SocketConnect/> 1`] = `"socket is connected"`;

exports[`<SocketConnect/> 2`] = `null`;

exports[`<SocketConnect/> 3`] = `null`;
